$NetBSD: patch-cmake_modules_CMakeLists.txt,v 1.4 2022/03/13 15:22:32 tnn Exp $

Make sure cmake rules go in own directory to avoid conflicts.

--- cmake/modules/CMakeLists.txt.orig	2021-09-24 16:18:10.000000000 +0000
+++ cmake/modules/CMakeLists.txt
@@ -1,6 +1,6 @@
 include(LLVMDistributionSupport)
 
-set(LLVM_INSTALL_PACKAGE_DIR lib${LLVM_LIBDIR_SUFFIX}/cmake/llvm)
+set(LLVM_INSTALL_PACKAGE_DIR share/libLLVM/cmake)
 set(llvm_cmake_builddir "${LLVM_BINARY_DIR}/${LLVM_INSTALL_PACKAGE_DIR}")
 
 # First for users who use an installed LLVM, create the LLVMExports.cmake file.
